main.go

```diff
@@ -16,12 +16,16 @@ import (
 
 // Input ...
 type Input struct {
-	Debug bool `env:"is_debug_mode,opt[yes,no]"`
+	Debug         bool   `env:"is_debug_mode,opt[yes,no]"`
+	BuildAPIToken string `env:"bitrise_build_api_token,required"`
```

Comment:
```suggestion
	BuildAPIToken stepconf.Secret `env:"bitrise_build_api_token,required"`
```
This type ensures that secret filtering masks the value, even if it leaks into the build log accidentally.